test/integration/tests/votium-bribe.ts

```diff
@@ -0,0 +1,104 @@
+
+    doLogging = Boolean(process.env.LOGGING);
+
+    const config = {
+      logging: doLogging,
+      deployAddress: deployAddress,
+      version: version
+    };
+
+    e2eCoord = new TestEndtoEndCoordinator(config, proposals);
+
+    doLogging && console.log(`Loading environment...`);
+    ({ contracts } = await e2eCoord.loadEnvironment());
+    doLogging && console.log(`Environment loaded.`);
+  });
+
+  before(async function () {
+    bribeSigner = await getImpersonatedSigner(FEI_BRIBE_ADMIN);
+  });
+
+  describe('When no voting round is active', async function () {
+    it('should revert if proposal is not found', async function () {
+      await expectRevert(contracts.votiumBriberD3pool.connect(bribeSigner).bribe(CVX_PROPOSAL, 44), 'invalid proposal');
+    });
+  });
+
+  describe('When a voting round is active', async function () {
+    before(async function () {
+      // start a new votium bribe round
+      const signer = await getImpersonatedSigner(VOTIUM_ADMIN);
+      await contracts.votiumBribe.connect(signer).initiateProposal(
+        CVX_PROPOSAL, // snapshot proposal id
+        Math.floor((Date.now() + 24 * 36e5) / 1000), // _deadline in 24h
+        50 // maxIndex in the proposal
+      );
+    });
+
+    it('should revert if index is out of boundaries', async function () {
+      await expectRevert(contracts.votiumBriberD3pool.connect(bribeSigner).bribe(CVX_PROPOSAL, 55), 'invalid choice');
+    });
+
+    it('should revert if caller does not have VOTIUM_BRIBE_ADMIN_ROLE role', async function () {
+      await expectRevert(
+        contracts.votiumBriberD3pool
+          .connect(await getImpersonatedSigner(VOTIUM_ADMIN)) // some address without role
+          .bribe(CVX_PROPOSAL, 42),
+        'CoreRef: Caller is not a governor or contract admin'
+      );
+    });
+
+    it('should revert if no tokens are available to bribe', async function () {
+      // empty potential TRIBE on the contract
+      const tribeBalance = await contracts.tribe.balanceOf(contracts.votiumBriberD3pool.address);
+      await contracts.votiumBriberD3pool.withdrawERC20(contracts.tribe.address, contracts.core.address, tribeBalance);
+
+      // bribe call should revert if there are no TRIBE to send
+      await expectRevert(
+        contracts.votiumBriberD3pool.connect(bribeSigner).bribe(CVX_PROPOSAL, 42),
+        'VotiumBriber: no tokens to bribe'
+      );
+    });
+
+    it('should allow VotiumBriber to bribe', async function () {
+      // Should receive a non-zero rewards stream from TribalChief
+      const briberBalanceBefore = await contracts.tribe.balanceOf(contracts.votiumBriberD3pool.address);
+      await contracts.stakingTokenWrapperBribeD3pool.harvest();
+      const briberBalanceAfter = await contracts.tribe.balanceOf(contracts.votiumBriberD3pool.address);
+      const bribeAmount = briberBalanceAfter.sub(briberBalanceBefore);
+      expect(bribeAmount).to.be.at.least('1');
```

Comment:
There should be a higher quality test here that does a bit more than just checking the bribe amount is at least 1. With current allocation points, it should be thousands of tribe if you have 250 AP